test.js

```diff
@@ -0,0 +1 @@
+test()
```

Comment:
What is this for?

Reply:
it was to see if files in root would be targetted